src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Answers {
+    private final List<Answer> answers = new ArrayList<>();
+
+    public Answers() {
+    }
+
+    public void addAnswer(Answer answer) {
+        this.answers.add(answer);
+    }
+
+    public List<DeleteHistory> deleteAnswers(NsUser loginUser) throws CannotDeleteException {
+        List<DeleteHistory> deleteHistories = new ArrayList<>();
+        for (Answer answer : answers) {
+            deleteHistories.add(answer.delete(loginUser));
+        }
+        return deleteHistories;
+    }
```

Comment:
```suggestion
    public List<DeleteHistory> deleteAnswers(NsUser loginUser) {
        return answers.stream()
                .map(answer -> answer.delete(loginUser))
                .collect(Collectors.toList());
    }
```

stream api 로 변경해 볼 수도 있겠네요.

Reply:
CannotDeleteException 을 꼭 선언해두어야하는 걸까요?
컴파일 타임에 throws CannotDeleteException 이 발생할까요?